.gitignore

```diff
@@ -21,6 +21,6 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
-.shift*
+.reshuffle*
 
-.vscode/
\ No newline at end of file
+.vscode/
```

Comment:
It seems like this is a file for some `$editor =~ /^v/`.  No editor-specific files in code, please.

Reply:
Cc @zolga: you can create a global gitignore instead of listing vscode in repos as described in https://help.github.com/en/articles/ignoring-files